staging/src/k8s.io/apiserver/pkg/storage/cacher/metrics/metrics.go

```diff
@@ -176,6 +176,29 @@ var (
 			Help:           "Counter for consistent reads from cache.",
 			StabilityLevel: compbasemetrics.ALPHA,
 		}, []string{"resource", "success", "fallback"})
+
+	WatchDispatchEventNumWatchers = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "watch_dispatch_event_num_watchers",
+			Help:           "Histogram of total number of watchers to dispatch event to.",
+			Buckets:        []float64{1, 2, 3, 4, 6, 8, 10, 15, 20, 30, 40, 50, 100, 200, 300, 500, 800, 1000, 2000, 3000},
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"resource"},
+	)
+
+	WatcherCountGauge = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "watcher_counter",
```

Comment:
just `watchers` would be a better name for the metric if we follow the Prometheus guidelines: https://prometheus.io/docs/practices/naming/

Reply:
Thanks, I'll rename it.